src/_tutorials/language/futures.md

```diff
@@ -59,47 +59,49 @@ Baseball score: Red Sox 10, Yankees 0
 ```
 
 Our code is problematic: since `gatherNewsReports()` blocks, the remaining
-code runs only when `gatherNewsReports()` returns with the contents of the
+code runs only after `gatherNewsReports()` returns with the contents of the
 file, _however long that takes_. And if reading the file takes a long time,
 the user waits passively, wondering if she won the lottery, what tomorrow's
 weather will be like, and who won today's game. Not good.
 
 To help keep the application responsive, Dart library authors use an
 asynchronous model when defining functions that do potentially expensive work.
-Such functions return their value using a [Future.][Future]
+Such functions return their value using a [`Future`.][Future]
 
-## What is a Future? {#what-is-a-future}
+## What is a future? {#what-is-a-future}
 
-A Future represents an asynchronous operation that returns a value (which
-can be `void`). When a function that returns a Future is invoked,
+A `Future` object represents an asynchronous operation that returns a value (which
+can be `void`). When a function that returns a future is invoked,
```

Comment:
[Ackshually](https://imgur.com/gallery/TKyAy), `void` isn't a value, it's a type. A `Future<void>` can theoretically complete to any value, you're just not allowed to use it.

How about:

> represents an asynchronous operation that may produce a value